db/createList.go

```diff
@@ -0,0 +1,18 @@
+package db
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+)
+
+func (d *dynamoDB) CreateList(listID *string) (error) {
+	input := &dynamodb.PutItemInput{
+		Item: map[string]*dynamodb.AttributeValue{
+			"Id": { S: listID }, "Name": { S: aws.String("Shopping List") },
```

Comment:
If you change `listID` that's passed in not to be a pointer, you will need to do `&listID` here

Reply:
Do we want to hard code `Shopping List` or should that be passed in as `listName` or something?